metron-deployment/vagrant/full-dev-platform/Vagrantfile

```diff
@@ -53,7 +53,7 @@ hosts = [{
 Vagrant.configure(2) do |config|
 
   # all hosts built on centos 6
-  config.vm.box = "bento/centos-6.7"
+  config.vm.box = "metron/centos_base"
```

Comment:
Exactly that for exactly that reason. That and I can control the swap space size now. I get a much more reliable (not perfect, but better) full dev experience with this image. Something that I wanted to do for quite some time, but I finally got it done because I was having trouble differentiating between errors I had introduced and pre-existing errors.